sql/moz-fx-data-shared-prod/monitoring/telemetry_missing_columns_v3/query.sql

```diff
@@ -0,0 +1,61 @@
+WITH placeholder_table_names AS (
```

Comment:
What is this CTE filtering out, exactly?

Reply:
There some schemas that have the placeholder schema in place, this omits them from being scanned. I'm not entirely sure if this is necessary, it's a leftover of the old query.